pkg/controller/.import-restrictions

```diff
@@ -252,7 +252,7 @@
         "k8s.io/kubernetes/pkg/capabilities",
         "k8s.io/kubernetes/pkg/master/ports",
         "k8s.io/kubernetes/pkg/scheduler/api",
-        "k8s.io/kubernetes/pkg/scheduler/util",
+        "k8s.io/kubernetes/pkg/scheduler/internal",
```

Comment:
Is this line needed? Was it auto-generated?

Reply:
I think it is autogenerated. Since I refactored util to internal, GoLand changed the dependency there. I am not sure if it is needed, but will check if it is required.

Reply:
We need this change. Just went through the dependencies.